Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionsView.swift

```diff
@@ -5,11 +5,36 @@
 import StreamChat
 import UIKit
 
+open class ChatReactionPickerReactionsView: ChatMessageReactionsView {
+    override public var reactionItemView: ChatMessageReactionItemView.Type {
+        components.reactionPickerReactionItemView
+    }
+}
+
 open class ChatMessageReactionsView: _View, ThemeProvider {
```

Comment:
Should we also change the name of the file to `ChatReactionPickerReactionsView.swift`?